drivers/led/gpio.c

```diff
@@ -0,0 +1,251 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <kernel.h>
+#include <led.h>
+#include <gpio.h>
+#include <device.h>
+#include <sys/util.h>
+#include <logging/log.h>
+
+LOG_MODULE_REGISTER(led_gpio, CONFIG_LEDS_GPIO_LOG_LEVEL);
```

Comment:
Logging is broken as I'm getting lot of warnings.

Reply:
I fix some issue with logging. I tested with debug log and without. You have to build sample for `nrf52840_pca10056` board. The DTS descriptions is only aligned for it in this PR.